pages/common/lstopo.md

```diff
@@ -0,0 +1,24 @@
+# lstopo
+
+> Show the hardware topology of the system.
+> More information: <https://manned.org/lstopo>.
+
+- Show the summarized system topology in a graphical window (or print to console if no graphical display is available):
+
+`lstopo`
+
+- Show the full system topology without summarizations:
+
+`lstopo --no-factorize`
+
+- Show the summarized system topology with only [p]hysical indices (i.e. as seen by the OS):
```

Comment:
The `[p]` is unnecessary because you used the full option name in the example.
```suggestion
- Show the summarized system topology with only physical indices (i.e. as seen by the OS):
```

Reply:
Are you sure about this? IIRC there were several PRs I reviewed in which this **was** the recommended style.

And a quick search (`rg -A2 --sort path '\[[\w\d]\][\w\d\-_]*'`) reveals quite a few examples of this:
- [cheatshh](https://github.com/tldr-pages/tldr/blob/7c4ebf35b654b930054964ad6906d1046e904701/pages/common/cheatshh.md)
- [cut](https://github.com/tldr-pages/tldr/blob/7c4ebf35b654b930054964ad6906d1046e904701/pages/common/cut.md)
- [d2](https://github.com/tldr-pages/tldr/blob/7c4ebf35b654b930054964ad6906d1046e904701/pages/common/d2.md)
- [du](https://github.com/tldr-pages/tldr/blob/7c4ebf35b654b930054964ad6906d1046e904701/pages/common/du.md)
- [git-status](https://github.com/tldr-pages/tldr/blob/7c4ebf35b654b930054964ad6906d1046e904701/pages/common/git-status.md)
- [katana](https://github.com/tldr-pages/tldr/blob/7c4ebf35b654b930054964ad6906d1046e904701/pages/common/katana.md)
- etc.

I don't have a strong opinion either way, but I think it's good to establish a consensus here.

Reply:
There are also pages like [azurite](https://github.com/tldr-pages/tldr/blob/7c4ebf35b654b930054964ad6906d1046e904701/pages/common/azurite.md), the style of which I think I actually prefer:

```md
- Use an existing [l]ocation as workspace path:

`azurite {{-l|--location}} {{path/to/directory}}`
```

Reply:
My mistake. I think we should leave it as it is, or do as in the case of `azurite`.